pages/common/fsck.md

```diff
@@ -0,0 +1,15 @@
+# fsck
+
+> Check the integrity of a filesystem or repair it.  The filesystem should be unmounted at the time the command is run.
```

Comment:
If you don't mind, please remove the double space between the sentences (for consistency with the other command pages).